include/menu.h

```diff
@@ -108,6 +116,8 @@ class Menu
         size_t index_menu_item_hover;
         // Whether a menu item is currently selected, so menu inputs should be forwarded to the MenuLine's handlers instead of navigating Menu
         bool is_menu_item_selected;
+        // Store current screen buffer (it will be transmitted over WiFi or Bluetooth)
+        char display_buffer[sizeof('\0') + (NUM_DISPLAY_LINES * (NUM_DISPLAY_CHARS_PER_LINE + sizeof('\n')))] = { 0 };
```

Comment:
No need for the `= { 0 }` here. I'm surprised it compiled.

Reply:
addressed in: https://github.com/jCallon/squirt/pull/1/commits/edb88a636d2cf54a42f970d190cd054c04993412